src/components/Forms.js

```diff
@@ -35,15 +37,25 @@ export const Input = props => {
     className,
     onChange,
     placeholder,
-    value
+    readOnly,
+    value,
+    icon
   } = props;
 
+  const styles = classNames({
+    [className]: className
```

Comment:
I'm fairly certain with classNames library if the argument is null it is not added. So you could just simply use `classNames(className)`

Reply:
👍

Reply:
@hudakdidit This gave me an error when I tried it :/

Reply:
What is the error? It says in the readme that falsey values are ignored. https://www.npmjs.com/package/classnames